packs/github/actions/add_team_membership.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: add_team_membership
```

Comment:
add_team_member perhaps?

Anyway, not a big deal, I'm also fine with the current name :)